src/tests/banners/Channel2BannerTests.ts

```diff
@@ -0,0 +1,60 @@
+import fetch from 'node-fetch';
+import {
+    BannerTemplate,
+    BannerTestGenerator,
+    BannerTest,
+    RawTestParams,
+    RawVariantParams,
+    BannerVariant,
+} from '../../types/BannerTypes';
+import { ContributionsBannerPath } from './ContributionsBannerTests';
+import { isProd } from '../../lib/env';
+
+export const DigitalSubscriptionsBannerPath = 'digital-subscriptions-banner.js';
+export const GuardianWeeklyBannerPath = 'guardian-weekly-banner.js';
+
+export const BannerPaths: { [key in BannerTemplate]: string } = {
+    [BannerTemplate.ContributionsBanner]: ContributionsBannerPath,
+    [BannerTemplate.DigitalSubscriptionsBanner]: DigitalSubscriptionsBannerPath,
+    [BannerTemplate.GuardianWeeklyBanner]: GuardianWeeklyBannerPath,
+};
+
+const Channel2BannerContentUrl = isProd
+    ? 'https://gu-contributions-public.s3-eu-west-1.amazonaws.com/banner/PROD/banner-tests2.json'
+    : 'https://gu-contributions-public.s3-eu-west-1.amazonaws.com/banner/CODE/banner-tests2.json';
+
+const Channel2BannerVariant = (variant: RawVariantParams): BannerVariant => {
+    return {
+        name: variant.name,
+        modulePath: BannerPaths[variant.template],
+        moduleName: variant.template,
+        bannerContent: {
+            messageText: variant.body,
+            heading: variant.heading,
+            cta: variant.cta,
+            secondaryCta: variant.secondaryCta,
+        },
+    };
+};
+
+const Channel2BannerTest = (testParams: RawTestParams): BannerTest => {
+    return {
+        name: testParams.name,
+        bannerType: 'subscriptions',
+        testAudience: testParams.userCohort,
+        locations: testParams.locations,
+        canRun: (): boolean => testParams.isOn,
+        minPageViews: testParams.minArticlesBeforeShowingBanner,
+        variants: testParams.variants.map(Channel2BannerVariant),
+        componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
+        articlesViewedSettings: testParams.articlesViewedSettings,
+    };
+};
+
+export const channel2BannersAllTestsGenerator: BannerTestGenerator = () =>
```

Comment:
Although for now we're keeping contribs and subs banners in separate lists, we will probably be mixing them soon.
So I think it's worth making this file more general and using it to replace the existing `ContributionsBannerTests.ts`. E.g.
```
const bannersFromToolGenerator: BannerTestGenerator = (fileName: string) => fetch(...

export const channel1BannersGenerator = bannersFromToolGenerator('banner-tests.json');
export const channel2BannersGenerator = bannersFromToolGenerator('banner-tests2.json');
```